helmfile.d/values/admin-namespaces-sc.yaml.gotmpl

```
@@ -0,0 +1,90 @@
+namespaces:
+  - name: cert-manager
+    labels:
+      pod-security.kubernetes.io/audit: restricted
+      pod-security.kubernetes.io/enforce: restricted
+      pod-security.kubernetes.io/warn: restricted
```

Do we want to add some slight templating in the chart to just be able to set:
```suggestion
namespaces:
  - name: cert-manager
    pss: restricted # or podSecurity or podSecurityStandard
```
maybe?

---

throwing the option of using yaml anchors in the mix as well

```yaml
pss:
  restricted:
    labels: &pss-restricted
      pod-security.kubernetes.io/audit: restricted
      pod-security.kubernetes.io/enforce: restricted
      pod-security.kubernetes.io/warn: restricted
  baseline:
    labels: &pss-baseline
      pod-security.kubernetes.io/audit: baseline
      pod-security.kubernetes.io/enforce: baseline
      pod-security.kubernetes.io/warn: baseline 
  restricted:
    labels: &pss-restricted
      pod-security.kubernetes.io/audit: restricted
      pod-security.kubernetes.io/enforce: restricted
      pod-security.kubernetes.io/warn: restricted

namespaces:
  - name: cert-manager
    labels: *pss-restricted
```

---

Leaving it as is for now